components/x-gift-article/src/RadioButtonsSection.jsx

```diff
@@ -11,7 +11,8 @@ const radioSectionClassNames = [
 ].join(' ');
 
 export default ({ shareType, showGiftUrlSection, showNonGiftUrlSection }) => (
-	<div className={ radioSectionClassNames }>
+	<div className={ radioSectionClassNames } role="group" aria-labelledby="article-share-options">
```

Comment:
I wonder if `role="group"` here is really necessary considering that is already on the parent. [See here](https://github.com/Financial-Times/x-dash/pull/482/files#diff-c19ca0f6bf98fd083196ea1f6a85f037R514). I don' remeber if need to be the immediate parent or not, maybe worth checking @thurstontye ?

Reply:
Will do but my understanding is they are different roles and so need to be declared separately.